app/views/articles/show.html.erb

```diff
@@ -9,3 +9,9 @@
   <strong>Text:</strong>
   <%= @article.text %>
 </p>
+
```

Comment:
A właściwie czemu to wszystko jest takie brzydkie?
Może dorzucimy jakiegoś bootstrapa?